test/apps/iodemo/io_demo.cc

```diff
@@ -722,7 +726,8 @@ class DemoServer : public P2pDemoCommon {
                           _prev_state.read_count << " ops, "
             << "write:" << _curr_state.write_count -
                            _prev_state.write_count << " ops, "
-            << "active connections:" << _curr_state.active_conns;
+            << "active connections:" << _curr_state.active_conns
+            << " buffers allocated:" << _data_buffers_pool.allocated();
```

Comment:
just "buffers: ", log line is already long

Reply:
fixed